system/jhtree/jhtree.cpp

```diff
@@ -1338,9 +1353,9 @@ void CKeyIndex::loadBloomFilters()
 
     while (bloomAddr)
     {
-        Owned<CJHTreeNode> node = loadNode(nullptr, nullptr, bloomAddr);
+        Owned<const CJHTreeNode> node = loadNode(nullptr, bloomAddr);
         assertex(node->isBloom());
-        CJHTreeBloomTableNode &bloomNode = *static_cast<CJHTreeBloomTableNode *>(node.get());
+        CJHTreeBloomTableNode &bloomNode = *(CJHTreeBloomTableNode *)node.get();
```

Comment:
minor: could this be const?

Reply:
Not at the moment (because get8 modifies). Might try to fix in my class structure refactor